File: Locations/MainBase/LivingQuarters.java
package Locations.MainBase;

import Locations.Location;
import People.Person;
import commandAndMessage.main.KeyConstants;

import java.util.Arrays;
import java.util.List;

/**
 * Created by Mark Chimes on 2017/11/09.
 */
public class LivingQuarters extends Location {

    public LivingQuarters(List<Person> allPeople, List<Location> sublocations) {
        super(allPeople, sublocations);
    }

    @Override
    public List<String> description() {
        return Arrays.asList("Description forthcoming");
    }

    @Override
    public String name() {
        return "Living Quarters";
    }

    @Override
    public List<String> nextMessage() {
        return KeyConstants.EMPTY_LIST;
    }



}

File: Locations/MainBase/LivingQuarters.kt
package Locations.MainBase

import Locations.Location
import People.Person
import commandAndMessage.main.KeyConstants

import java.util.Arrays

/**
 * Created by Mark Chimes on 2017/11/09.
 */
class LivingQuarters(allPeople: List<Person>, sublocations: List<Location>) : Location(allPeople, sublocations) {

    override fun description(): List<String> {
        return Arrays.asList("Description forthcoming")
    }

    override fun name(): String {
        return "Living Quarters"
    }

    override fun nextMessage(): List<String> {
        return KeyConstants.EMPTY_LIST
    }


}
